keyboards/darkproject/kd87a_bfg_edition/info.json

```diff
@@ -0,0 +1,110 @@
+{
+    "keyboard_name": "KD87A_BFG_Edition",
+    "manufacturer": "GSKY",
+    "url": "",
+    "maintainer": "Gsky",
+    "usb": {
+        "vid": "0x342D",
```

Comment:
https://www.the-sz.com/products/usbid/index.php?v=0x342D&p=&n=

This appears to be the VID for Westberry.  Is this board sold by westberry?

If not, 0xFEED works, but otherwise, needs to be one that isn't in use, and the further past 0x4000, the better.

Reply:
we apply VID with westberry. Westberry gave this VID which isn't used by others.

Reply:
Okay. Would appreciate verification, if possible.